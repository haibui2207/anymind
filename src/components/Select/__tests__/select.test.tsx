import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import CustomSelect from "../Select";

describe("Testing Select component", (): void => {
  const props = {
    name: "select",
    onChange: jest.fn(),
    options: [
      { label: "Option 1", value: 1 },
      { label: "Option 2", value: 2 },
    ],
  };

  it("should render match snapshot", (): void => {
    const wrapper = shallow(<CustomSelect {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
