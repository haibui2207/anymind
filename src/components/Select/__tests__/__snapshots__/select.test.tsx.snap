// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Select component should render match snapshot 1`] = `
<ForwardRef
  className="select-0-2-1"
  classNamePrefix="custom-react-select"
  components={
    Object {
      "DropdownIndicator": [Function],
    }
  }
  instanceId="select"
  isDisabled={false}
  isSearchable={false}
  name="select"
  onChange={[Function]}
  options={
    Array [
      Object {
        "label": "Option 1",
        "value": 1,
      },
      Object {
        "label": "Option 2",
        "value": 2,
      },
    ]
  }
/>
`;
